dev/io.openliberty.security.jakartasec.3.0.internal.cdi/src/io/openliberty/security/jakartasec/cdi/beans/OidcHttpAuthenticationMechanism.java

```diff
@@ -254,7 +254,18 @@ private AuthenticationStatus processCallback(Client client, HttpMessageContext h
         try {
             ProviderAuthenticationResult providerAuthenticationResult = client.continueFlow(request, response);
             status = processContinueFlowResult(providerAuthenticationResult, httpMessageContext, client);
-        } catch (UnsupportedResponseTypeException | AuthenticationResponseException | TokenRequestException e) {
+        } catch (AuthenticationResponseException e) {
+            Tr.error(tc, e.getMessage());
```

Comment:
Please move the Tr.error under the INVALID_RESULT case. A NOT_VALIDATED_RESULT does not need a Tr.error because the request could simply be for another resource that just happens to have a state parameter.

Thanks!